docling_core/types/doc/base.py

```diff
@@ -59,27 +59,72 @@ def scaled(self, scale: float) -> "BoundingBox":
         :param scale: float:
 
         """
-        out_bbox = copy.deepcopy(self)
-        out_bbox.l *= scale
-        out_bbox.r *= scale
-        out_bbox.t *= scale
-        out_bbox.b *= scale
-
-        return out_bbox
+        warnings.warn(
+            "scaled is deprecated: use `scale_to_size` instead.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self.scale_to_size(size=Size(width=scale, height=scale))
 
     def normalized(self, page_size: Size) -> "BoundingBox":
         """normalized.
 
         :param page_size: Size:
 
         """
-        out_bbox = copy.deepcopy(self)
-        out_bbox.l /= page_size.width
-        out_bbox.r /= page_size.width
-        out_bbox.t /= page_size.height
-        out_bbox.b /= page_size.height
+        warnings.warn(
+            "normalized is deprecated: use `normalize_to_size` instead",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self.normalize_to_size(size=page_size)
+
+    def normalize_to_size(self, size: Size) -> "BoundingBox":
+        """normalize_to_size.
+
+        :param page_size: Size:
+
+        """
+        return BoundingBox(
+            l=self.l / size.width,
+            r=self.r / size.width,
+            t=self.t / size.height,
+            b=self.b / size.height,
+            coord_origin=self.coord_origin,
+        )
 
-        return out_bbox
+    def scale_to_size(self, size: Size) -> "BoundingBox":
```

Comment:
scale_to_size is unnecessary since scaled can accommodate this, and also a misnomer. If anything it would be scale_by_size.